ionoscloud/data_source_certificate_manager_auto_certificate.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
-	certSDK "github.com/ionos-cloud/sdk-go-cert-manager"
+	certSDK "github.com/ionos-cloud/sdk-go-bundle/products/cert/v2"
```

Comment:
certSDK should be renamed to certsdk. Camelcase is not allowed for package names.

Reply:
@cristiGuranIonos done in dd8991f053c3efdc4b729b223b188320381e65c2